"Id","CaseNumber","Ticket_Record_Type_Name__c","Subject","Description","DevOps_Steps_to_Reproduce__c","What_We_Know__c","Next_Steps__c","Solution_Comment__c","Issue_Type__c","Ticket_Owner_Name__c","Ticket_Type__c","Platform__c","Collector_Code__c","Feature__c","Product_Version__c","Build_Number__c","Age__c","Status","Account_Name__c","Contact_Name__c","CreatedDate","ClosedDate"
"500Qk00000KLw8kIAD","00431428","Support Ticket","Cannot update to v5.9.4.1 - Checksum error","hi,
I tried to udpate from 5.9.4.0 to 5.9.4.1 and I received Error 5, some details details with checksum error, different than the Live Update.
I waited couple of minutes and then checked again for updates and now the error is saying that ""Number of maximum retries reached, skipping this update..""
thanks,
Daniel","","","","Removed error from the backend","","Alin Anca","Error","Netwrix Endpoint Protector","Server","Server LiveUpdate","","","1.0","Closed - Resolved","HQLAx","Daniel HQLAx IT Support","2025-01-08T09:38:38.000Z","2025-01-09T09:20:11.000Z"
